website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -60,6 +61,29 @@ For example, given the forward-to URL `https://my-custom-app.com/api/123`, the `
 - `forward-url-full-path`: `https://my-custom-app.com/api/123`
 - `forward-url-origin`: `https://my-custom-app.com`
 
+## Configuring custom claims
+
+This header can be used to make the Merchant Center API Gateway to include extra public claims in the exchange token sent along with forwarded requests.
+
+The value must be a list of the claims separated by an empty space.
+
+The following claims are supported:
+* **permissions**: Optionally includes the list of applied permissions for the logged in user. The permission values are defined in the format `can<permission-name>`, for example `canViewAvengers`.
+
+```
+"X-Forward-To-Claims": "permissions"
+```
+
+<Info>
+
+This feature is available from version `21.14.0` onwards.
+
+</Info>
```

Comment:
Nit: since we are using a bullet list, we should indent these lines so that they appear correctly indented below the bullet point.

````suggestion

  ```
  "X-Forward-To-Claims": "permissions"
  ```

  <Info>

  This feature is available from version `21.14.0` onwards.

  </Info>
````

Reply:
Updated in this commit: 97804574d2708408fb60e045faf4f4fdb091ee6f